extensionsGallery-insider.json

```diff
@@ -4316,6 +4316,59 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "85",
+					"extensionName": "net-core-3-runtime",
+					"displayName": ".NET Core 3.1 Runtime",
+					"shortDescription": "The .NET Core 3.1 Runtime extension downloads runtime binaries that can be used by other extensions.",
+					"publisher": {
+						"displayName": "Microsoft",
+						"publisherId": "Microsoft",
+						"publisherName": "Microsoft"
+					},
+					"versions": [
+						{
+							"version": "1.0.13",
+							"lastUpdated": "9/16/2021",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+									"source": "https://dsct.blob.core.windows.net/extensions/net-core-3-runtime/21259.2/net-core-3-runtime-1.0.13.vsix"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+									"source": "https://dsct.blob.core.windows.net/extensions/net-core-3-runtime/21259.2/icon.png"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
+									"source": "https://dsct.blob.core.windows.net/extensions/net-core-3-runtime/21259.2/README.md"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Code.Manifest",
+									"source": "https://dsct.blob.core.windows.net/extensions/net-core-3-runtime/21259.2/package.json"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.License",
+									"source": "https://dsct.blob.core.windows.net/extensions/net-core-3-runtime/21259.2/LICENSE.rtf"
+								}
+							],
+							"properties": [
```

Comment:
Other extensions have link to the source code in this section, but we do not publish sources for our extensions, so I removed it. Let me know if it's required and I should keep it here with empty value or just pointing to github.com/Microsoft.

Reply:
This particular extension can be (and probably will be) open-sourced in the future. There are no real blockers, aside from extended compliance process that we didn't go through yet. Having said that, it should not be a blocker, as other extensions that we are building will not be open-sourced for business reasons and we should be able to accommodate that.

Bugs are expected to come through CSS, "repo for submitting bugs" is not mandatory for a release of Microsoft product, but on-boarding with CSS is. I've also placed our feedback email alias in the README, so that customers can reach to us directly.

Reply:
I guess just linking to the Microsoft org seems fine for now then